docs/index.md

```diff
@@ -39,17 +42,29 @@ The image below shows the timing around how the Mobi Planner turns Bookings and
 ## Amazon Web Services (AWS) and API Interfaces
 
 - **Inputs:** [Bookings and Flights](#bookings-and-flights) are streamed to Mobi via an AWS Kinesis Data Stream
+  - Jamie: The way this will work is still kind of an open question...  Under the covers I think that we will want to
+    own our own Kinesis stream for this--but I'm not sure exactly how we want to expose this to them.  One relevant
+    question is: does CyberLogic themselves use AWS?
```

Comment:
I sent them an email asking, will let you know @jcgentry what they say. I was under the impression though that we would push Kinesis for future clients? I guess if they do something different on their end its their choice. Do you feel its too soon to say that?